lucene/core/src/java/org/apache/lucene/codecs/compressing/Lucene90CompressingTermVectorsFormat.java

```diff
@@ -33,7 +33,7 @@
  *
  * @lucene.experimental
  */
-public class CompressingTermVectorsFormat extends TermVectorsFormat {
+public class Lucene90CompressingTermVectorsFormat extends TermVectorsFormat {
```

Comment:
I think all these classes should be moved under the package `lucene90.compressing`. That way all of our "versioned" format files (plus their helper classes) will be in a package with the version name, ready to move to backwards-codecs when needed. This will be consistent with #2310.

Reply:
I will do that once we move Lucene87StoredFieldFormat